src/provider/model/ConfigStore.ts

```diff
@@ -0,0 +1,30 @@
+import {Store} from 'openfin-service-config';
+import {Loader} from 'openfin-service-config/Loader';
+import {injectable} from 'inversify';
+
+import {ConfigurationObject} from '../../../gen/provider/config/fdc3-config';
+import {AsyncInit} from '../controller/AsyncInit';
+
+@injectable()
+export class ConfigStore extends AsyncInit {
+    private _store: Store<ConfigurationObject>;
+    private _loader: Loader<ConfigurationObject>;
+
+    constructor() {
+        super();
+        this._store = new Store(require('../../../gen/provider/config/defaults.json'));
+        this._loader = new Loader(this._store, 'fdc3');
```

Comment:
Could this just be `new Loader(this._store, 'fdc3');`?

Reply:
Yes, thats how I originally had it.  The change was made because we would have no way to access it from devtools for any debugging if needed